### ISSUE TITLE
Using the "--json" flag can output incompatible json.

### ISSUE THREAD

**Issue by tomkinsc** [conda/conda#2799]
### What's happening:

When I run `conda search biopython --json` I see:

```
Using Anaconda Cloud api site https://api.anaconda.org
{
  "biopython": [
    {
      "build": "np15py26_0",
      "build_number": 0,
       [...]
```

### Why this is not ideal:
Since the conda command prints "`Using Anaconda Cloud api site https://api.anaconda.org`" to stdout, the output cannot be interpreted directly as json, and the "`Using Anaconda Cloud[...]`" line before the json object must be removed before parsing what follows as json.

### Proposed solution:
Omit printing `Using Anaconda Cloud api site https://api.anaconda.org` when the `--json` flag is specified (for this command, and all others where it occurs).

### Additional information:

Output of `conda info`:

```
Using Anaconda Cloud api site https://api.anaconda.org
Current conda install:

             platform : osx-64
        conda version : 4.1.2
  conda-build version : 1.21.0
       python version : 3.5.1.final.0
     requests version : 2.9.1
     root environment : /Users/tomkinsc/miniconda3  (writable)
  default environment : /Users/tomkinsc/miniconda3
     envs directories : /Users/tomkinsc/miniconda3/envs
        package cache : /Users/tomkinsc/miniconda3/pkgs
         channel URLs : https://conda.anaconda.org/t/<TOKEN>/bioconda/osx-64/
                        https://conda.anaconda.org/t/<TOKEN>/bioconda/noarch/
                        https://repo.continuum.io/pkgs/free/osx-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/osx-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : /Users/tomkinsc/.condarc
    is foreign system : False
```

**Comment by kalefranz** [comment]
Completely agree.  Hopefully will be resolved in conda 4.2.

**Comment by kalefranz** [comment]
Confirmed, this will be fixed in 4.2.0 as part of #2537.
